### ISSUE TITLE
Model related by hasOne relation doesn't update via relation manager

### ISSUE THREAD

**Issue by BadLame** [octobercms/october#5353]
- PHP Version: 7.4.12
- Plugins Installed: October.Test-v1.0.3

### Description:
Model related by hasOne relation doesn't update via relation manager

### Steps To Reproduce:
1) Define hasOne relation in model
2) Display this field for model in backend using relation manager
3) Create connected model and save it
4) Try to update recently created model through relation manager
5) After save - refresh page

### Expected result
- After updates apply (click "Update" in the modal window) - corresponding field values changed
- After model save (click "Save" in the main window) and page refresh - related model fields values still changed

### Actual result
- After updates apply (click "Update" in the modal window) - corresponding field values changed
- After model save (click "Save" in the main window) and page refresh - related model fields values resets to the values __before update__

**Comment by LukeTowers** [comment]
What version of October are you using and how did you install it?

**Comment by BadLame** [comment]
Version: 469
Install: composer create-project ...

**Comment by zaqsk** [comment]
Its bug in october core:
Just change line 1160 in `modules\backend\behaviors\RelationController.php`

Line 1160:

`$this->viewModel = $this->manageWidget->model;`
to this:

```
// Ensure that the view widget model is the same instance as the manage widget model
// since they will technically be different object instances in this context as
// $viewWidget->model is populated by $this->relationModel->getResults() and
// $manageWidget->model is populated by $this->relationModel->find($manageId);
$this->viewModel = $this->viewWidget->model = $this->manageWidget->model;
```

**Comment by bennothommo** [comment]
@BadLame @zaqsk This bug is fixed in October CMS 1.1.0 and above. You will need to update to those versions to take advantage of it.
